addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayDeviceHandler.java

```diff
@@ -260,9 +260,9 @@ public void run() {
                             refreshChannel(channel);
                         } catch (Throwable t) {
                             if (t instanceof Exception) {
-                                logger.error("Error occurred when performing polling:" + ((Exception) t).getMessage());
+                                logger.error("Error occurred when performing polling:{}", t.getMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace